webview/src/plots/components/comparisonTable/ComparisonTableWrapper.tsx

```diff
@@ -11,7 +11,7 @@ export const ComparisonTableWrapper: React.FC = () => {
   const { size, isCollapsed } = useSelector(
     (state: PlotsState) => state.comparison
   )
-  const handleResize = (size: PlotSize) => {
+  const handleResize = (size: number) => {
```

Comment:
[C] Seems like there are three versions of the same `handleResize` wrapper function.

Reply:
It's because they are dispatching 3 different actions (they share the same name, but are imported from different files).